server/tests/centers.js

```diff
@@ -0,0 +1,376 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import app from '../app';
+import helper from './testHelper';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+chai.should();
+faker.seed(300);
+
+module.exports = describe('Tests for the centers', () => {
```

Comment:
All the comments I made below also apply here.
Finally, you can write unit tests  to ensure that your models are well tests too.